### ISSUE TITLE
size of gap/patch in ED?

### ISSUE THREAD

**Issue by fabeit** [EDmodel/ED2#137]
I am confused about the actual spatial size of a gap/patch in ED. Given that I am running a simulation with only one met grid cell that is 100x100 km and in only one site. I read in Medvigy 2009 that " Each grid cell is subdivided into a series of tiles. The relative area of
each tile is determined by the proportion of canopy-gap sized
areas within the grid cell having a similar canopy structure as
a result of a common disturbance history."

What does this actually mean? Is a tile a patch?How does ED differ from a regular gap model in which  one gap is the size of a tree crown, about 15x15m?

**Comment by mdietze** [comment]
Yes, what David's calling a tile is a patch. ED differs from a regular gap model in that the patches are not laid out on some giant checkerboard, but rather are completely _implicit_ in space (they don't have locations themselves). Instead, each patch is treated as representative of some fraction of the overall landscape, thus it has a fractional area as an attribution (what David called the relative area in the quote above). Nominally, ED patches are conceived of as being analogous in size to traditional gap models (though there's not an exact specification of size), but then the dynamics of each is expanded out across the larger landscape using the fractional area as a weight.

**Comment by fabeit** [comment]
Thanks I understand the bit better but I'm still confused about a few aspects. For example say I have two simulations:1. Grid of 100x100km and 2. Grid 10x10km, say they both have only one patch. In the case of 1 that patch represents 100km2 and for simulation 2 is a 10km2 patch? Can the patch in simulation 1 contain more cohort/plants because it represents a bigger area? Will it be more computationally expensive?

How does ed extrapolate from patches with a relative area an absolute value of plants/m2 and all the other extensive values?

**Comment by mdietze** [comment]
Short answer: there's no connection to the overall grid cell area and the patch areas. The number of patches and cohorts used is controlled by MAXPATCHES and MAXCOHORTS which has no relation to the grid area.  All units in ED are per unit area based so if you want to extrapolate to the total at a regional scale you have to know the area yourself and multiply by it.

Longer answer: I'd argue that ED is not running at a point, like some models do -- when we run on a grid we're running the box not the intersection, but everything within the box is implicit (no space). ED is built on the assumption that you are working at a large enough spatial scale that everything can be treated as distributions. If you run ED at too small of a spatial scale that assumption breaks down and you dive into the realm of spatially explicit landscape models rather than ED's spatially implicit approach.
